### PR TITLE
Feature/#139456269/czar draws card by clicking

### PR DESCRIPTION
**Pull request by freemile2ru**
# What does this PR do?

Add pop up prompting the czar that he is the czar
Add czar draw card functionality
Upload the questions and answers to the DB
Animate the deck of cards

# How should this be manually tested?
by playing the game from the review app and observing this new features

# Any background context you want to provide?
this PR made several file changes and tried to modify most codes in files touched to follow eslint rules

# What are the relevant pivotal tracker stories?
CZARs should be able to draw cards by clicking

# Screenshots (if appropriate)
N/A
# Questions:
N/A

### CODE REVIEW

**Review by eshaibu — DISMISSED** [review]

**Review by eshaibu — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -7,5 +7,4 @@ public/lib
 .DS_store
 c4h_keys.txt
 coverage
-.env
```
Comment (andela-ooduye) [inline .gitignore]: Why did you get rid of the .env from here? The env file is never to be pushed!

File: app/views/includes/foot.jade [hunk 2]
```diff
@@ -3,7 +3,8 @@ script.
       (function(d,t){var g=d.createElement(t),s=d.getElementsByTagName(t)[0];
       g.src='//www.google-analytics.com/ga.js';
       s.parentNode.insertBefore(g,s)}(document,'script'));
-script(type='text/javascript', src='/lib/jquery/jquery.js')
+//script(type='text/javascript', src='/lib/jquery/jquery.js')
```
Comment (ooduye) [inline app/views/includes/foot.jade]: Why do we have commented out code?

File: app/views/includes/foot.jade [hunk 3]
```diff
@@ -3,7 +3,8 @@ script.
...
+script(type='text/javascript', src='https://code.jquery.com/jquery-2.2.4.js')
```
Comment (ooduye) [inline app/views/includes/foot.jade]: Why are you using a CDN? When the application is created, it already gets the latest version of jquery in the link you commented above

File: bower.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
     "bootstrap": "~3.0.0",
     "angular-bootstrap": "~0.4.0",
     "angular-ui-utils": "0.0.4",
-    "jquery": "~1.9.1",
+    "jquery": "~2.1.4",
```
Comment (ooduye) [inline bower.json]: Any reason for this change?

File: config/socket/game.js [hunk 5]
```diff
@@ -166,90 +180,99 @@ Game.prototype.stateChoosing = function(self) {
...
     this.players[winnerIndex].points++;
     this.winnerAutopicked = true;
     this.stateResults(this);
+    // czar to select card
+    this.sendNotification(`${this.players[winnerIndex].username} has won the round!`);
+    this.sendUpdate();
   } else {
-    // console.log(this.gameID,'no cards were picked!');
-    this.stateChoosing(this);
+//    // console.log(this.gameID,'no cards were picked!');
```
Comment (ooduye) [inline config/socket/game.js]: Remove all commented out code

File: config/socket/game.js [hunk 6]
```diff
@@ -166,90 +180,99 @@ Game.prototype.stateChoosing = function(self) {
...
+//    this.stateChoosing(this);
```
Comment (ooduye) [inline config/socket/game.js]: Remove all commented out code

**Review by ooduye — CHANGES_REQUESTED** [review]
Review these changes and correct them ASAP

**Review by ooduye — APPROVED** [review]

**Pull request merged**